python/lsst/ts/MTMount/mock/controller.py

```diff
@@ -362,6 +509,93 @@ async def put_deployable_telemetry(self, device_id, tai):
             )
         )
 
+    async def put_motion_controller_state(self, system_id, tai):
+        """Output MOTION_CONTROLLER_STATE, if changed.
+
```

Comment:
A gentle suggestion to also document the method arguments.

Reply:
Good suggestion. Done for all the put methods.

Reply:
In cleaning this up I found a bunch more mock.Controller methods with no docs, so I added documentation for those as well.